package.json

```diff
@@ -17,11 +17,10 @@
   },
   "requires": true,
   "dependencies": {
-    "request-promise": "^4.2.6"
+    "axios": "^0.21.1"
```

Comment:
Should also increment the sample version inside this file as well as `package-lock.json`

Reply:
versions update in the package.json and package-lock.json files